molecule/cookiecutter/scenario/driver/docker/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/INSTALL.rst

```diff
@@ -1,10 +1,11 @@
 *******
-Install
+Docker driver installation guide
```

Comment:
If we are going to change this heading.  Could you change the heading for all cookiecutter drivers in
`molecule/cookiecutter/scenario/driver/*/\{\{cookiecutter.molecule_directory\}\}/\{\{cookiecutter.scenario_name\}\}/INSTALL.rst`

Reply:
ok, done.